tests/BoundingBoxTest.php

```diff
@@ -42,17 +44,15 @@ public function testConstructorShouldRequireAnEvenNumberOfValues(): void
         new BoundingBox([0, 0, 1, 1, 2]);
     }
 
-    /**
-     * @dataProvider provideBoundsWithInvalidTypes
-     */
-    public function testConstructorShouldRequireIntegerOrFloatValues(): void
+    #[DataProvider('provideBoundsWithInvalidTypes')]
+    public function testConstructorShouldRequireIntegerOrFloatValues(mixed ...$args): void
```

Comment:
Contrasting this with PointTest, I'm curious why you used the splat operator here instead of receiving a single `$bounds` argument. That would require wrapping the arguments in the data provider with an extra array (similar to what you did in PointTest).

`$bounds` is more self-documenting than `$args` IMO, as I think most readers would expect you to unpack `$args` when invoking the BoundingBox constructor.